README.rst

```diff
@@ -39,15 +39,27 @@ Additional functionality provided by `ftw.subsite`
 Compatibility
 -------------
 
-Plone 4.3
+Plone 4.3 + Dexterity + Migration
+
+With ``ftw.subsite`` 2.x only the AT based Subsite content is replaced with a
+Dexterity based Subsite content + ftw.simplelayout default page. Thus the
+Subsite portlet column, column renderer and the teaser portlet are no longer used and will be removed with ``ftw.subsite`` 2.1.0. The ``ftw.subsite`` release 2.0.x will provide a inplace migration. Be aware only static text portlets and subsite teaser portlets will be migrated. You need to provide your own migration for other portlets.
+The subsite_view will be available in ``ftw.subsite`` 2.0.x and removed in 2.1.x.
```

Comment:
As discussed with @maethu this morning: please add a note about the compatibility and the requirements (`plone.app.contenttypes` and `plone.app.event`).